react/src/lib/components/GroupTree/components/Settings/DateTimeSlider.test.tsx

```diff
@@ -3,25 +3,27 @@ import { render, screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import "jest-styled-components";
 import React from "react";
-import { testStore, Wrapper } from "../../test/TestWrapper";
+//import { testStore, Wrapper } from "../../test/TestWrapper";
```

Comment:
Suggest cleaning up disabled code.

Reply:
Tenkte å kanskje ta det i neste steg ettersom jeg ikke vet helt hvordan de virker.

Reply:
@hkfb : I have updated the tests so that they don't break.
@nilscb : I have pushed a commit to your PR to solve the test related issues. Please have a look.